docs/configuration/cron_jobs.rst

```diff
@@ -85,6 +85,19 @@ You can also limit the number of Contacts to process per script execution using
 
     that these messages are only added to the queue when frequency rules apply either system wide or per Contact.
 
+.. vale off
+
+Custom Field chron jobs
+=======================
+
+.. vale on
+
+**To keep contacts and company custom fields updated**
```

Comment:
```suggestion
**To create contacts and company custom fields in the background:** The custom fields are created when you submit the form to create a custom field by default. However, this operation can take longer than the HTTP request time limit allows on bigger Mautic instances. If you get into this point, add this row to your app/config/local.php file:

``'create_custom_field_in_background' => true,``

This will stop creating the custom fields on form submission and instead adds the custom fields via a command that will be run by cron which has no time limits set.
```